assets/prototype/application/ui/components/display/espressoIcon/isEspressoIcon.ts

```diff
@@ -0,0 +1,19 @@
+import { Icon } from './types';
+
+/**
+ * returns true if icon is an EspressoIcon
+ */
+const isEspressoIcon = (icon: string): boolean => {
```

Comment:
I am thinking if we can use just enums.

Reply:
right... will do...
the only catch is that I still have to allow `string` as a default type, because the intent was to allow `EspressoIcon` to also be a wrapper for WP Dashicons where we could add additional styles or add other tweaks if need be

Reply:
Good point, wasn't aware of that :+1: